buildkite_agent.rb

```diff
@@ -0,0 +1,23 @@
+dep "buildkite-agent" do
+  requires "buildkite-agent.managed"
+  requires "buildkite-agent.launchctl"
+end
+
+dep "buildkite-agent.managed", :buildkite_token do
+  requires "homebrew tap".with "buildkite/buildkite"
+
+  met? {
```

Comment:
@timriley I couldn't easily see how to get the `install` method that can be used with the managed template to take the `--devel` option, so I used the custom `meet` block, which I then paired for consistency with the `met?` block. But you're right, it doesn't need it - I can remove.